openslides_backend/action/actions/mediafile/upload.py

```diff
@@ -90,18 +90,24 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         file_ = instance.pop("file")
         decoded_file = base64.b64decode(file_)
         mimetype_ = python_magic.from_buffer(decoded_file, mime=True)
+        use_mimetype: Optional[str] = mimetype_
```

Comment:
Change sequence of lines 93 and 94. Then you could use type `str` for the `use_mimetype` and there is no need for casting in line 125

Reply:
Won't work because it may be filled with None in line 97